roborock/version_1_apis/roborock_local_client_v1.py

```diff
@@ -121,20 +110,56 @@ async def async_disconnect(self) -> None:
         async with self._mutex:
             self._sync_disconnect()
 
-    async def hello(self):
+    def _set_encoder_decoder(self):
+        """Updates the encoder decoder. For L01 these are updated with nonces after the first hello."""
```

Comment:
This is now always true that it updates with the ack nonce after hello, but still worth preserving the fact that this is needed for L01